www/src/components/forum/Feedback.tsx

```diff
@@ -45,11 +66,20 @@ export function Feedback() {
       return;
     }
 
-    const feedbackData = {
-      ...data,
+    posthog.capture("USER_REACTED_NEGATIVELY_TO_SUMMARY", {
```

Comment:
It would be great to see feedback on this 👇. I have no experience with posthog analysis, so could you guys tell me what json content format would make future analysis easier?

Reply:
I was also wondering if we should have an event for every time a user selects a specific category